Разметка внутренней страницы

maket-2 opened the pull request:
---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/1999673/tasks/3)

---
:mortar_board: [Разметка внутренней страницы](https://up.htmlacademy.ru/htmlcss/33/user/1999673/tasks/4)

maket-2 committed:
index

Разметка главной страницы

maket-2 committed:
изменение главной

Изменение футора главной страницы

maket-2 committed:
Выполнена страница каталога

Разметка страницы каталога

maket-2 committed:
Изменение название страницы

catalog.html

4err commented on catalog.html:
```diff
@@ -0,0 +1,253 @@
+          </a>
+        </div>
+        <nav class="navigation">
+          <ul class="navigation-list">
```
Часть классов написаны через дефис, часть через нижнее подчеркивание. Нужно привести к единому виду.

4err commented on catalog.html:
```diff
@@ -0,0 +1,253 @@
+    <header class="page_header">
+      <div class="wrapper">
+        <div class="header_logo">
+          <a href="" class="header_logo_link">
```
Логотип ведет на главную страницу

4err commented on catalog.html:
```diff
@@ -0,0 +1,253 @@
+            <li class="navigation-item">
+              <label class="header_login">
+                 <input type="text" class="form-control" name="query" placeholder="Текст поиска" autocomplete="off">
+                 Найти</label>
```
Для работы инпута необходимо добавить форму.

4err commented on catalog.html:
```diff
@@ -0,0 +1,253 @@
+                 Найти</label>
+            </li>
+            <li class="navigation-item">
+                  <a class="login-buttom" href="">Коля Петухов</a>
```
Форматирование кода. Здесь и далее. Нужно привести к единому виду.

4err replied:
Форматирование все еще ездит. и тут и далее.

4err commented on catalog.html:
```diff
@@ -0,0 +1,253 @@
+          </ul>
+          <ul class="header-catalog-list">
+            <li class="navigation-item">
+              <a href="catalog.html">Каталог товаров</a>
```
Мы сейчас уже находимся на странице `catalog.html`. Нужно избегать создания цикличных ссылок страниц на самих себя. Если элемент является ссылкой, но в какой-то момент он не является ею, потому что, например, ведет на саму себя, мы убираем ему атрибут href. Так браузер и поисковики поймут, что данный элемент, хоть сейчас и не ссылка, но когда-то он ею является.
Нужно исправить здесь и далее.

4err commented on catalog.html:
```diff
@@ -0,0 +1,253 @@
+                <img class="product-card-image" src="" alt="Любительская селфи-палка">
+                <h3 class="product-card-title">Любительская селфи-палка</h3>
+              </a>
+              <b class="product-card-price">500 ₽</b>              
```
Каждая карточка, вполне самостоятельный блок. Мы можем засунуть ее в слайдер, на главную, в рекламу. Поэтому их следует обернуть в `article`

4err commented on catalog.html:
```diff
@@ -0,0 +1,253 @@
+              <b class="product-card-price">1500 ₽</b>              
+            </li>            
+          </ul>
+          <button class="button pagination-show-more" type="button">Загрузить еще</button>
```
Это тоже ссылка

4err commented on index.html:
```diff
@@ -5,8 +5,277 @@
+          </li>
+          <li>
+              <input type="radio" id="property-3" name="property-three">
+              <label for="slide3"></label>
```
Блок управления слайдами нужно вынести отдельно, он не привязан к непосредственным слайдам

4err commented on index.html:
```diff
@@ -5,8 +5,277 @@
+          </li>          
+      </ul>
+    </div>
+    <div class="slaider">
```
slider же

4err commented on index.html:
```diff
@@ -5,8 +5,277 @@
+          </button>
+        </li>
+      </ol>
+    </div>
```
А вот изображения должны быть привязаны к слайдам. Стрелки переключают не изображение в рамках слайда, а сами слайды. Соответственно у тебя каждое изображение должно быть в слайдах, а стрелки также нужно вынести в отдельный блок управления к точкам.

4err commented on index.html:
```diff
@@ -5,8 +5,277 @@
+        </li>
+        <li class="delivery-item-link">
+          <a class=delivery-link href="#"><img src="" alt=""></a>
+        </li>
```
Здесь список ни к чему, это просто несколько соответствующих блоков.

4err reviewed (changes requested):
На будущее, не смешивай задания, их нужно выполнять друг за другом. 
Одинаковые ошибки в разных файлах не повторяю, все актуально и там и там. 
Отличная работа!

maket-2 committed:
Исправление главной

Изменено:
1. Классы приведены к единому виду.
2. Добавлена форма для поиска
3. Выведен отдельно переключение слайдером
4. Блок быстрая доставка изменена на отдельные блоки

maket-2 committed:
Изменение страницы каталога

1. Приведение классов к общему виду
2. Обернуты article карточки товара
3. Изменен синтаксис кнопки загрузить еще
4. Убрана цикличная ссылка

4err commented on index.html:
```diff
@@ -5,8 +5,254 @@
+    <div class="swiper-pagination">
+    <input class="slider-radio" id="1" type="radio" checked>
+    <input class="slider-radio" id="2" type="radio">
+    <input class="slider-radio" id="3" type="radio">
```
Так у тебя же есть уже кнопки. Они же будут и отражать страницы. Зачем этот блок?

4err commented on index.html:
```diff
@@ -5,8 +5,254 @@
+      <ul class="catalog-categoriest-list">
+        <li class="catalog-categories-item">
+          <img src="" alt="Виртуальная реальность">
+          <a class="catalog-categories-link" href="">Виртуальная реальность</a>
```
По стайлгайду изображение также является ссылкой.

4err reviewed (changes requested):
Поехало форматирование местами. Причеши код в соответствии с кодгайдом:
http://codeguide.academy/html-css.html

maket-2 committed:
Исправление главной

Устранение замечаний

maket-2 committed:
Исправление страницы каталога

Устранение замечаний страницы каталога

4err commented on catalog.html:
```diff
@@ -0,0 +1,262 @@
+              <a class="navigation-link" href="">Гарантия</a>
+            </li>
+            <li class="navigation-item">
+              <a class="navigation-link" href="">Контакты</a>
```
закрывающая лишка потерялась.

maket-2 committed:
исправление главной

устранение замечаний

maket-2 committed:
исправление страницы каталога

Устранение замечаний li и _

4err reviewed (changes requested):
https://take.ms/YHa9G

https://take.ms/BobSi

Все еще форматирование. Открывающий и закрывающий теги должны быть на одной вертикали. горизонтальный отступ должен быть одинаковым: 2 пробела. Посмотри внимательно еще раз кодгайд и код.

maket-2 committed:
форматирование главной

maket-2 committed:
форматирование каталога
